test-platform-ui/app/lib/editor/richtext.tsx

```diff
@@ -0,0 +1,100 @@
+import {
```

Comment:
This should be place in the Atoms folder, lib folder should be contain the configuration only

Reply:
Relocated folder, thanks @sts-nghianguyen